Integrations/integration-Atlassian_Confluence_Server.yml

```diff
@@ -0,0 +1,650 @@
+        headers = {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json'
+        }
+        full_url = BASE_URL + '/space'
+        res = requests.get(full_url, verify=VERIFY_CERTIFICATE, auth=(USERNAME,PASSWORD), headers=headers, params=params)
+        if res.status_code < 200 or res.status_code >= 300:
+            return_error_and_exit('Failed to list Spaces.\nURL: {}, Status Code: {}, Response: {}'.format(full_url, res.status_code, res.text))
+        return res.json()
+
+    def list_spaces_command():
+        """
+        Confluence list Spaces method
+        """
+        space_list = list_spaces()
+        spaces = []
+        for raw_space in space_list['results']:
+            space = {}
+            space['ID'] = raw_space['id']
+            space['Key'] = raw_space['key']
+            space['Name'] = raw_space['name']
+            spaces.append(space)
+
+        # create markdown table string from context
+        # the outputs must be array in order the tableToMarkdown to work
+        # headers must be array of strings (which column should appear in the table)
+        md = tableToMarkdown('Spaces', spaces, ['ID', 'Key', 'Name'])
+
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['json'],
+            'Contents': spaces,
+            'HumanReadable': md,
+            'EntryContext': {
+                'Confluence.Space': spaces
+            }
+        })
+
+    def delete_content(content_id):
+        headers = {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json',
+        }
+        full_url = BASE_URL + '/content/' + content_id
+        res = requests.delete(full_url, verify=VERIFY_CERTIFICATE, auth=(USERNAME,PASSWORD), headers=headers)
+        result = {}
+        if res.status_code == 204:
+            result['Results'] = "Successfully Deleted Content ID " + content_id
+            result['ID'] = content_id
+        else:
+            return_error_and_exit('Delete failed. Check Content ID.\nURL: {}, Status Code: {}, Response: {}'.format(full_url, res.status_code, res.text))
+        return result
+
+    def delete_content_command():
+        """
+        Confluence Delete Content Spaces method
+        """
+
+        content_id = demisto.args().get('id')
+
+        deleted_content = delete_content(content_id)
+
+        # create markdown table string from context
+        # the outputs must be array in order the tableToMarkdown to work
+        # headers must be array of strings (which column should appear in the table)
+        md = tableToMarkdown('Content', deleted_content['results'], ['Result', 'ID'])
+
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['json'],
+            'Contents': deleted_content,
+            'HumanReadable': md,
+            'EntryContext': {
+                'Confluence.Content': deleted_content
```

Comment:
dt, and i would add a Confluence.Content.deleted value

Reply:
done.